apps/site/assets/js/algolia-result.js

```diff
@@ -259,7 +261,8 @@ function getTransitIcons(hit) {
   }
   const icons = iconFromGTFS(
     hit.related_transit_gtfs_id,
-    hit.related_transit_gtfs_ancestry
+    hit.related_transit_gtfs_ancestry,
+    hit._highlightResult.route.name.value
```

Comment:
```suggestion
    hit.related_transit_gtfs_ancestry
```

I don't see any changes to `iconFromGTFS` to have it accept a third argument so I'm presuming this was left here mistakenly.